components/fxa-client/src/http_client.rs

```diff
@@ -894,4 +951,164 @@ mod tests {
         };
         assert_eq!("{\"grant_type\":\"refresh_token\",\"client_id\":\"bar\",\"refresh_token\":\"foo\",\"scope\":\"bobo\",\"ttl\":123}", serde_json::to_string(&using_code).unwrap());
     }
+
+    #[test]
+    fn test_backoff() {
```

Comment:
thank you for creating these tests ! 🏅

Reply:
Tbh was kinda surprised that we don't have tests for everything else in this file 🙃